### PR TITLE
Adds the Potion Launcher

### PR DESCRIPTION
**Pull request by Irradiation**
Long overdue (I said I'd do this about 2 months ago).

Coming at relatively cheap cost (5 spell points), the potion launcher is meant to be used for rapid firing of potions. It comes with 5 free murky potions and can hold up to 5 potions in its loader and also comes with a silly new sound effect when it fires. It also can be attached to the suit slot of wizard robes and gem hardsuits.

Sprites courtesy of the now-dead Terrel is Trait.
Sound courtesy of RealestEstate.
capture
potionzooker

[gamemode][balance][sound][sprites]

:cl:
 * rscadd: Added the Potion Launcher artifact for wizards. It comes with 5 free murky potions, it can hold up to 5 potions and buying it costs 5 spell points.
<!--
Pull requests must be atomic.  Change one set of related things at a time.  Bundling sucks for everyone.
This means, primarily, that you shouldn't fix bugs and add content in the same PR. When we mean 'bundling', we mean making one PR for multiple, unrelated changes.

Test your changes. PRs that do not compile will not be accepted.
Testing your changes locally is incredibly important. If you break the serb we will be very upset with you.

Large changes require discussion.  If you're doing a large, game-changing modification, or a new layout for something, discussion with the community is required as of 26/6/2014.  Map and sprite changes require pictures of before and after.  MAINTAINERS ARE NOT IMMUNE TO THIS.  GET YOUR ASS IN IRC.

Merging your own PRs is considered bad practice, as it generally means you bypass peer review, which is a core part of how we develop.

It is also suggested that you hop into irc.rizon.net #vgstation or the coding discord (you can find an invite link on the irc or the thread) to discuss your changes, or if you need help.

== SELF LABELLING PRs ==
You can now self-label your PR! The syntax is simple. Just put [<labeltag>] anywhere in your PR,
where <labeltag> is to be replaced by one of the following (allowing with the resultant tag)
just ctrl+f the list or something. I know it's long.

administration = "Logging / Administration"
away = "Mapping (Away Mission :earth_africa:)"
bagel = "Mapping (Bagel :o:)"
box = "Mapping (Box :baby:)"
bugfix = "Bug / Fix"
bus = "Mapping (Bus :bus:)"
byond = "T-Thanks BYOND"
consistency = "Consistency Issue"
controversial = "Controversial"
deff = "Mapping (Deff :wastebasket:)"
discussion = "Discussion"
dnm = "✋ Do Not Merge ✋"
easy = "Easy Fix"
exploitable = "Exploitable"
featureloss = "Feature Loss"
featurerequest = "Feature Request"
first = "good first issue"
formatting = "Grammar / Formatting"
gamemode = "Gameplay / Gamemode"
gameplay = "Gameplay / Gamemode"
general = "Mapping (General :world_map:)"
goonchat = "Goonchat"
grammar = "Grammar / Formatting"
help = "help wanted"
hotfix = "Hotfix"
libvg = "libvg"
logging = "Logging / Administration"
meta = "Mapping (Meta :no_mobile_phones:)"
needspritework = "Needs Spritework"
oversight = "Oversight"
packed = "Mapping (Packed :package:)"
parallax = "Parallax"
qol = "❤️ Quality of Life ❤️"
roid = "Mapping (Roidstation :pick:)"
role = "Role Datums"
roleissue = "Role Datums Issue"
runtime = "Runtime Log"
sanity = "Sanity / Ghosthands"
snowmap = "Mapping (Snowmap ❄)"
sound = "Sound"
sprites = "Sprites"
spriteworkdone = "Spritework Done Needs Coder"
system = "System"
taxi = "Mapping (Taxi :taxi:)"
tested = "100%  tested"
tweak = "Tweak"
ui = "UI"
vault = "Mapping (Vault :question:)"
vote = "⛔ Requires Server Vote ⛔"
wip = "WiP"

== CHANGELOGS ==
Changelogs can be attached either by following the instructions in html/changelogs/example.yml, or by attaching an in-body changelog like the one attached to your PR automatically.

For the keys you can use in an in-body changelog, they are the same as described in example.yml
NOTE that anything *after* the :cl: will be parsed as a changelog, if it somehow manages to be parseable as such, so always put the changelog at the VERY end!

### CODE REVIEW

**Comment by Kurfursten** [comment]
Should probably come with some freebie potions since it is literally worthless on its own

**Comment by Irradiation** [comment]
I was considering making it cost 0 points because potions are mostly garbo compared to pretty much everything else in the Wizman's kit, but I'm sure 5 murky potions wouldn't be too bad to accompany it.

**Comment by boy2mantwicethefam** [comment]
ok but does it play the ice barrage sound when fired

**Comment by Irradiation** [comment]
Not yet, they don't.

File: code/modules/projectiles/guns/projectile/potion_launcher.dm [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+	if((istype(I, /obj/item/potion)))
+		if(potions.len < max_potions)
+			if(user.drop_item(I, src))
+				potions += I
+				to_chat(user, "<span class='notice'>You load the [I.name] into the [src.name].</span>")
+				to_chat(user, "<span class='notice'>[potions.len] / [max_potions] potions loaded.</span>")
+		else
+			to_chat(user, "<span class='warning'>The [src.name] cannot hold more potions.</span>")
+
```
Comment (MadmanMartian) [inline code/modules/projectiles/guns/projectile/potion_launcher.dm:36]: May want to return 1 if the item is a potion, else the potions will call afterattack, which may cause you to have a flaming, sword-impaled, potion launcher

**Comment by Kurfursten** [comment]
This PR got so old, the absolute @MadmanMartian added ice barrage!
